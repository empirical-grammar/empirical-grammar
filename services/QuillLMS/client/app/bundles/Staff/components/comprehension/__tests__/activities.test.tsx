import * as React from 'react';
import { shallow } from 'enzyme';
import { DataTable } from 'quill-component-library/dist/componentLibrary';
import Activities from '../activities';
import 'whatwg-fetch';

describe('Activities component', () => {
  const container = shallow(<Activities />);

  it('should render Activities', () => {
    expect(container).toMatchSnapshot();
  });
});
