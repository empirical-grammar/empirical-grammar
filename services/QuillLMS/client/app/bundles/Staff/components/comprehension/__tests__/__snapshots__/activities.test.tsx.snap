// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Activities component should render Activities 1`] = `
<div
  className="activities-container"
>
  <DataTable
    averageFontWidth={7}
    className="activities-table"
    defaultSortAttribute="title"
    headers={
      Array [
        Object {
          "attribute": "title",
          "name": "Title",
          "width": "700px",
        },
        Object {
          "attribute": "course",
          "name": "Course",
          "width": "300px",
        },
      ]
    }
    rows={Array []}
  />
</div>
`;
